src/psycop_feature_generation/application_modules/flatten_dataset.py

```diff
@@ -74,15 +74,17 @@ def create_flattened_dataset(
     Returns:
         FlattenedDataset: Flattened dataset.
     """
-    filtered_prediction_times_df = filter_prediction_times(
-        prediction_times_df=prediction_times_df,
-        project_info=project_info,
-        quarantine_df=quarantine_df,
-        quarantine_days=quarantine_days,
-    )
+
+    if quarantine_df or quarantine_days:
+        prediction_times_df = filter_prediction_times(
```

Comment:
```suggestion
        filtered_prediction_times_df = filter_prediction_times(
```

Having two different variable names makes it possible to inspect their diff during debugging, at only a small cost in RAM useage.